lib/auth/grpcserver.go

```diff
@@ -618,6 +618,11 @@ func (g *GRPCServer) GenerateUserCerts(ctx context.Context, req *authpb.UserCert
 	}
 
 	if req.Purpose == authpb.UserCertsRequest_CERT_PURPOSE_SINGLE_USE_CERTS {
+		if req.MFAResponse != nil {
+			if _, ok := req.MFAResponse.Response.(*authpb.MFAAuthenticateResponse_TOTP); ok {
+				return nil, trace.BadParameter("per-session MFA is not satisfied by 'otp' devices, please update your configuration to use WebAuthn")
```

Comment:
Where does the convention of calling them `'otp' devices` comes from? Could this term  be replaced with `OTP devices`?

Reply:
I think it came from a warning message where we quoted `"u2f"`, but I'm not sure why I carried that forward here 🤔 Fixed!